Active Documents/contenteditable-disabled.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset='utf-8'>
+    <title>ContentEditable-Disabled</title>
+    <script class='remove'>
+        var respecConfig = {
+            specStatus: "ED",
+            editors: [
+                {
+                    name: "Johannes Wilm",
+                    mailto: "johannes@fiduswriter.org",
+                    company: "Invited Expert"
+                },
+                {
+                    name: "Bo Cupp",
+                    mailto: "pcupp@microsoft.com",
+                    company: "Microsoft"
+                },
+                {
+                    name: "Grisha Lyukshin",
+                    mailto: "glyuk@microsoft.com",
+                    company: "Microsoft"
+                }
+            ],
+            github: "https://github.com/w3c/editing",
+            shortName: "contenteditable-interface",
+            wg: "Web Applications Working Group",
+            wgURI: "https://www.w3.org/2019/webapps/",
+            wgPatentURI: "https://www.w3.org/2004/01/pp-impl/114929/status",
+            license: "w3c-software-doc"
```

Comment:
We get the right license by default, but we need to enable "xref", for cross references.

```suggestion
            xref: "web-platform", // we will be citing html, infra, and friends.
```

Reply:
similarly to the issues above, I've changed it on my end.